modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js

```diff
@@ -141,13 +141,13 @@ function downloadAgent() {
         values[this.name] = $(this).val();
     });
 
-    var connectedCupRegisterURL = "/connectedcup_mgt/connectedcup/cup/register?" +
-                                                        "name=" + payload.name + "&owner=" + payload.owner;
-
     var payload = {};
     payload.name = $inputs[0].value;
     payload.owner = $inputs[1].value;
```

Comment:
Use double quoted object key access for reading JSON attributes as it's considered to be more reliable.
Ex: person.name as person["name"]